MailCore/Models/SwissTransferAttachment.swift

```diff
@@ -0,0 +1,39 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+import MailResources
+import RealmSwift
+
+public class SwissTransferAttachment: EmbeddedObject, Codable {
+    @Persisted public var uuid: String
+    @Persisted public var nbfiles: Int
+    @Persisted public var size: Int64
```

Comment:
Why should we precise Int64? As Swift Int are already 64 bits

Reply:
No particular reason, just to be consistent with the Attachment model